dart/lib/src/sentry_stack_trace_factory.dart

```diff
@@ -13,6 +13,13 @@ class SentryStackTraceFactory {
   static const _stackTraceViolateDartStandard =
       'This VM has been configured to produce stack traces that violate the Dart standard.';
 
+  static const _sentryPackagesIdentifier = <String>[
+    'sentry',
```

Comment:
I've noticed that the other packages aren't checked, so I've also added this.

Reply:
Could this use registered `options.sdk.packages`? (`pub:sentry_logging`)

Reply:
Yeah, probably. @marandaneto how do you handle this in other SDKs?

Reply:
Not really, the `sdk.packages` are the identifier that connects with https://github.com/getsentry/sentry-release-registry, Our docs read this repo to get the latest versions, Sentry UI also to see if you're not using outdated packages